.appveyor.yml

```diff
@@ -10,18 +10,15 @@ clone_folder: c:\mill
 environment:
   matrix:
   - COMPILER: default
-    JAVA_HOME: C:\Program Files\Java\jdk9
+    JAVA_HOME: C:\Program Files\Java\jdk1.8.0
```

Comment:
What's this change for?

Reply:
I changed the msys2 env to test dev.assembly, which requires changing to Java 9. I think it's good to test mill on both Java 8 and 9 in Windows so I changed the default one to Java 8.